charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -15,8 +15,8 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-713.service.core-compute-preview.internal/
-    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-224.service.core-compute-preview.internal/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-aat.service.core-compute-aat.internal/
+    USER_PROFILE_HOST: http://ccd-user-profile-api-aat.service.core-compute-aat.internal/
```

Comment:
you'd also need to list the PR number in jenkinsfile_cnp for the FTs to be wired with AAT.